extra_tests/snippets/stdlib_urllib_https_misaligned_recv.py

```
@@ -0,0 +1,246 @@
+        pos = 0
+        while pos < len(response):
+            size = plaintext_sizes.pop(0) if plaintext_sizes else 16384
+            end = min(len(response), pos + size)
+            while pos < end:
+                try:
+                    pos += tls.write(response[pos:end])
+                except ssl.SSLWantWriteError:
+                    pass
+                drain_outgoing(outgoing, conn)
+        conn.close()
+    except BaseException as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `BaseException`

(BLE001)